dts/arm/st/stm32f446-pinctrl.dtsi

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2017 Pushpal Sidhu
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dt-bindings/pinctrl/stm32-pinctrl.h>
+
+/ {
+	soc {
+		pinctrl: pin-controller {
+			usart2_pins_b: usart2@1 {
+				rx_tx {
+					rx = <STM32_PIN_PD6 (STM32_PINMUX_ALT_FUNC_7 | STM32_PUPDR_NO_PULL)>;
+					tx = <STM32_PIN_PD5 (STM32_PINMUX_ALT_FUNC_7 | STM32_PUSHPULL_NOPULL)>;
+				};
+			};
+			usart3_pins_c: usart3@2 {
+				rx_tx {
+					rx = <STM32_PIN_PC11 (STM32_PINMUX_ALT_FUNC_7 | STM32_PUSHPULL_PULLUP)>;
+					tx = <STM32_PIN_PC10 (STM32_PINMUX_ALT_FUNC_7 | STM32_PUSHPULL_PULLUP)>;
+				};
+			};
+			usart6_pins_a: usart6@0 {
```

Comment:
usart6 not supported, please remove

Reply:
Not supported in Zephyr? Just wondering.. but why not?

Reply:
My bad, usart6 is actually supported on stm32f446. Discard this comment.